src/olympia/ratings/tests/test_admin.py

```diff
@@ -257,6 +257,30 @@ def test_search_by_ip(self):
         assert addon.guid in addon_filter_options_text
         assert second_addon.guid in addon_filter_options_text
 
+        # Sort by IP address using django admin built-in sort:
+        # parameter is `o`, value is -5.3 because we're sorting by IP (the 5th
```

Comment:
Wow, that's some opaque data formatting for the sort parameter.

Reply:
Yeah... My guess is that's a super old legacy thing that never got changed... It's weird that it's not stable if columns ordering or contents change.